docs/docs/07-faq.md

```diff
@@ -0,0 +1,187 @@
+# Tips, Pointers, and FAQ
+
+We do our best to make our documentation easy to navigate; however, LittleHorse has a lot of features and it may not always be easy to find what you are looking for.
+
+Think of this page as an "index" that will help you find what you need to make your project a success.
+
+## Workflow Basics
+
+Read here for how to get started.
+
+### What is a `WfSpec` and `WfRun`?
+
+The core concept of LittleHorse is a [Workflow](./04-concepts/01-workflows.md), which defines a series of tasks to execute and events to wait for. A `WfSpec` defines a workflow specification. A `WfRun` is a running instance of that `WfSpec`.
+
+### How do I create a `WfSpec`?
+
+Check out our [`WfSpec` development docs](./05-developer-guide/08-wfspec-development/08-wfspec-development.md)! In short, you can use our Java, Python, or Go SDK's to define the `WfSpec` logic and then register them to the LH Server.
+
+### What is a Task Worker?
+
+A [Task Worker](./04-concepts/03-tasks.md) is a program that opens a connection to a LittleHorse Cluster and listens to a task queue for a specific `TaskDef`. When a `WfRun` arrives at a point where it needs to execute that specific type of `TaskRun`, then the scheduled Task will be dispatched to the Task Worker. The Task Worker executes it and reports the result to the LH Server.
+
+### How do I write a Task Worker?
+
+Check out our [Task Worker Development Docs](./05-developer-guide/05-task-worker-development.md)!
+
+### How do I run a Workflow?
+
+You can run a workflow in two ways:
+* On the command line by [using `lhctl`](./05-developer-guide/03-lhctl.md#run-a-wfrun)
+* Programmatically by [using one of our SDK's](./05-developer-guide/09-grpc/10-running-workflows.md).
```

Comment:
* Programmatically by [using one of our SDK's](./05-developer-guide/09-grpc/10-running-workflows.md).  In the background the SDKs call a GRPC endpoint hosted on the LittleHorse server.

Reply:
Programmatic execution is ideal for incorporating into your dashboards, and operational tools.